### ISSUE TITLE
Update EC2 instances with latest packages from OS repos

### ISSUE THREAD

**Issue by frankhereford** [cityofaustin/atd-data-tech#11834]
use yum to update system packages on all 5 EC2 instances.

comment with list of ec2 instances that were updated.

-- 
snapshot each machine so we can rollback if necessary

**Comment by frankhereford** [comment]
We discussed the interval that we want to do this operation, and there was agreement that matching the update interval that CTM uses for the on-prem servers is an appropriate balance.

**Comment by frankhereford** [comment]
@roseeichelmann This came to both of us so we can pair on it and both pick up the experience of doing these upgrades. Spreading out the knowledge of system administration tasks for team resiliency. I'm thinking it will take an hour tops, maybe more like 30 minutes. It should be as easy as using `yum` to see what needs to be installed and doing it. Thanks!

**Comment by frankhereford** [comment]
@roseeichelmann and I were able to pair on this this morning. We got a lot done:

* All 6 ec2 instances (now, 5) are running the latest and greatest system packages.
* the postgrest MDS machine has been imaged and terminated
* We discussed the sizing on the m5.large machine
* Rose has SSH access and sudo rights on all machines

Everything we'd hoped to get done we were able to, so I'm going to close this. Thank you @roseeichelmann!!
